e2e/fn/eval/simple-function/.expected/config.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
```

Comment:
Pl. have `testdata` directory under e2e. If we want to reduce nesting, may be have `fn-eval` as directory.

Reply:
Moved to `e2e/testdata/fn-eval`.